lm_eval/api/model.py

```diff
@@ -169,6 +183,12 @@ def world_size(self):
         # not support multi-device parallelism nor expect it.
         return self._world_size
 
+    @property
+    def tokenizer_name(self) -> str:
+        raise NotImplementedError(
```

Comment:
```suggestion
        """Must be defined for LM subclasses which implement Chat Templating. Should return the name of the tokenizer or chat template used. Used only to properly fingerprint caches when requests are being cached with `--cache_requests`, otherwise not used."""
        raise NotImplementedError(
```